storage/gc_queue_test.go

```diff
@@ -316,8 +316,8 @@ func TestGCQueueIntentResolution(t *testing.T) {
 
 	// Two transactions.
 	for i := 0; i < 2; i++ {
-		// TODO(spencer): increase once batching support is available.
-		for j := 0; j < 50; j++ {
+		// 5 (TODO(spencerkimball): 50,000) puts per transaction.
```

Comment:
attempt at an inline comment (5 now, TODO for spencer: change to 50k). I'll make it clearer.